aten/src/ATen/native/README.md

```diff
@@ -277,6 +277,18 @@ them the same thing!)
 If two backends have the same dispatch function, you can write `CPU, CUDA: func`
 to reuse the same function name in both cases.
 
+TODO: expand this section.
+
+Available backend options can be found at
+https://github.com/pytorch/pytorch/blob/master/tools/codegen/gen.py#L970.
+In addition to backends above, we also support keyword `Math` which is an alias
+that maps to all backends. In other words, function registered to `Math` key
+should be a plain mathematical composition of other `at::` functions and works for any backend.
+
+If you add `dispatch` section to any API that didn't have it before, you **have to** move
+the old implementation to `Math` field so that it's still available for other backends to use.
```

Comment:
We can also enforce this, right? (Maybe it's further down the PR...)

Reply:
Good point! We currently cannot enforce it easily since we don't track whether ops has dispatch sections and whether they change. But this should be possible in the terminal state where we can track this through our API provided to backend extensions. ;) So I'd save this as a followup task. ;)

Reply:
Ah yeah - actually I oversimplifying , picturing a world where we could simply insist that if you had any explicit dispatch entries at all, then `Math` had to be one of them (say, by adding a semantic check in `NativeFunction.from_yaml`). But of course we'd need a codemod (or something) before our current declarations would fail that test, so yeah - followup task sounds right!